src/EnergyPlus/CurveManager.cc

```diff
@@ -2087,12 +2086,23 @@ namespace CurveManager {
                         ShowContinueError("    b) no normalization reference values are defined.");
                         ErrorsFound = true;
                     } else if (pointsSpecified) {
-                        // normalizeGridValues normalizes to 1.0 at the reference values. We must redivide by passing in the 1.0/normalizationDivisor as the scalar here.
-                        btwxtManager.normalizeGridValues(gridIndex, PerfCurve(CurveNum).GridValueIndex, normalizeTarget, 1.0/normalizationDivisor);
+                        // normalizeGridValues normalizes curve values to 1.0 at the normalization target, and returns the scalar needed to perform this normalization.
+                        // The result is multiplied by the input normalizationDivisor again for the AutomaticWithDivisor case, in which normalizeGridValues returns a compound scalar.
+                        normalizationDivisor = btwxtManager.normalizeGridValues(gridIndex, PerfCurve(CurveNum).GridValueIndex, normalizeTarget, normalizationDivisor)*normalizationDivisor;
```

Comment:
@mjwitte The issue here is that the data going into btwxt has already been normalized by the divisor when lookupValues is set (around line 2059). Btwxt then does the autonormalization for the target based on the normalized results, then applies the scalar.

For example, for a single table value of 6 at the normalization target and a user-divisor of 3:
- The value is normalized by the user-divisor when it is read in: 6/3 = 2
- The value is then autonormalized to 1 in Btwxt making the auto-divisor = 2
- Btwxt then applies the user-divisor passed into the `normalizeGridValues` function: 1/3 = 0.333
- `normalizeGridValues` then returns its auto-divisor times the user-divisor: 2*3 = 6
- The actual divisor (between user input and the value used in Btwxt) is: 6/0.333 = 18
- To get to the actual divisor from what Btwxt returns you need to multiply the value from `normalizeGridValues` by the user-divisor: 18 = 6*3

I know this is convoluted, and maybe there is a better way to go about it by adding more functionality to btwxt, but there are complications related to when the btwxt objects are created and when normalization is performed.

Would it help to include this description in the comment?

Reply:
@nealkruis "The issue here is that the data going into btwxt has already been normalized by the divisor when lookupValues is set (around line 2059)." Ah, that's the key. That prior normalization seems premature, but so be it. Not holding this up for that. But if you make any other changes to this, a comment here that points to the earlier normalization would help. Or, better yet, seems it would be more clear to skip the earlier normalization and do it here.